models/ad_reporting_models.yml

```diff
@@ -143,6 +143,7 @@ models:
           combination_of_columns:
             - platform
             - date_day
+            - keyword_id
             - keyword_text
```

Comment:
should we be testing on _just_ keyword_id?

Reply:
worded that weird -- i mean should we remove keyword_text from the test

Reply:
Yeah I think it would be appropriate to remove the keyword_text. And would be in-line with updates from the upstream packages.